File: app/src/main/java/trunk/doi/base/ui/fragment/product/mvp/ProductModel.java
package trunk.doi.base.ui.fragment.product.mvp;


import com.base.lib.integration.repository.IRepositoryManager;
import com.base.lib.mvp.BaseModel;

import com.base.lib.di.scope.FragmentScope;

import javax.inject.Inject;

import trunk.doi.base.ui.fragment.product.mvp.ProductContract;

/**
 * ================================================
 * Description:
 * <p>
 * Created by GasMvpFragment on 11/30/2019 14:55
 * ================================================
 */

@FragmentScope
public class ProductModel extends BaseModel implements ProductContract.Model {

    @Inject
    public ProductModel(IRepositoryManager repositoryManager) {
        super(repositoryManager);
    }

    @Override
    public void onDestroy() {
        super.onDestroy();

    }
}

File: app/src/main/java/com/gas/app/ui/fragment/product/mvp/ProductModel.kt
package com.gas.app.ui.fragment.product.mvp

import com.base.lib.di.scope.FragmentScope
import com.base.lib.integration.repository.IRepositoryManager
import com.base.lib.mvp.BaseModel
import javax.inject.Inject

/**
 * ================================================
 * Description:
 *
 *
 * Created by GasMvpFragment on 11/30/2019 14:55
 * ================================================
 */
@FragmentScope
class ProductModel @Inject constructor(repositoryManager: IRepositoryManager) : BaseModel(repositoryManager), ProductContract.Model {
    override fun onDestroy() {
        super.onDestroy()
    }
}